label_studio/tasks/models.py

```diff
@@ -234,6 +234,16 @@ def prepare_filename(filename):
             return filename.replace(settings.MEDIA_URL, '')
         return filename
 
+    def resolve_storage_uri(self, url, project):
+        if not self.storage:
+            storage_objects = project.get_all_storage_objects(type_='import')
+            self.storage = self._get_storage_by_url(url, storage_objects)
+
+        if self.storage:
+            return self.storage.generate_http_url(url)
+
+        return null
```

Comment:
null is not allowed, you should either remove this return at all or use None, this line will get 500 error